### PR TITLE
Feature/at 91/2 d bigger mega emoji

### PR DESCRIPTION
**Pull request by aminbksi**
https://momentum.nifty.pm/l/LWO5xQxL72M

Mega emoji size position

Mute button access for admins only

### CODE REVIEW

**Review by PAKovalev — APPROVED** [review]

File: packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const EmojiAnimationDock: FC = () => {
           (emojiMsg && (
             <EmojiWithAvatarAnimation key={idx} emojiMsg={emojiMsg} offset={offset} />
           )) ||
-          (emojiUrl && <MegamojiAnimation key={idx} emojiUrl={emojiUrl} offset={offset} />) || <></>
+          (emojiUrl && <MegamojiAnimation key={idx} emojiUrl={emojiUrl} />) || <></>
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx:56]: You don't want to use randomisation on X-axis position anymore?

File: packages/app/src/scenes/collaboration/components/EmojiAnimationDock/components/MegamojiAnimation/MegamojiAnimation.styled.ts [hunk 2]
```diff
@@ -2,42 +2,45 @@ import styled from 'styled-components';
 
 export const MegamojiContainer = styled.div`
   position: absolute;
-  top: 100%;
+  height: 220px;
+  width: 220px;
   opacity: 0;
-  width: 150px;
-  height: 150px;
+  top: 30%;
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/collaboration/components/EmojiAnimationDock/components/MegamojiAnimation/MegamojiAnimation.styled.ts:8]: I'm sure it's working but it probably needs to be tested on big and small display.

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx (hunk 1 above)
Comment (aminbksi) [reply packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx:56]: Yeah basically for now it's somehow in the centre, but we'll test tomorrow on Dev, to see if it is a good idea to have it there.

File: packages/app/src/scenes/collaboration/components/EmojiAnimationDock/components/MegamojiAnimation/MegamojiAnimation.styled.ts (hunk 2 above)
Comment (aminbksi) [reply packages/app/src/scenes/collaboration/components/EmojiAnimationDock/components/MegamojiAnimation/MegamojiAnimation.styled.ts:8]: Well I did change my display size and it works fine.

File: packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx (hunk 1 above)
Comment (dmitry-yudakov) [reply packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx:56]: It was less predictable before, but that's fine this way too :)

Comment (aminbksi) [reply packages/app/src/scenes/collaboration/components/EmojiAnimationDock/EmojiAnimationDock.tsx:56]: yeah that's true, only for mega emojis tho, I was thinking of initiate in right position and ending it up in centre, but we keep in in centre for now to see what're the feedbacks.

**Pull request merged**